### PR TITLE
Add component documentation

### PR DESCRIPTION
**Pull request by CaldwellYSR**
**Components documented:**
- Match Center X
- Match Center Y
- Tiled Properties

### CODE REVIEW

File: lib/components/match-center-x.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/** Set this entity's X position based on the X position of another entity.
+ * @typedef {object} id
```
Comment (ericlathrop) [inline lib/components/match-center-x.js]: `id` should probably be `matchCenterX`

File: lib/components/match-center-y.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/** Set this entity's Y position based on the Y position of another entity.
+ * @typedef {object} id
```
Comment (ericlathrop) [inline lib/components/match-center-y.js]: `id` should probably be `matchCenterY`

**Comment by CaldwellYSR** [comment]
Then the component would be `matchCenterY: { "matchCenterY": 1 }`.

I can make that change if you want.

**Comment by ericlathrop** [comment]
Don't change the component, the doc comment just looked wrong. I'll test it.

**Comment by ericlathrop** [comment]
I tested it, and the doc comments _are_ wrong. The component name just shows up as `id`.

**Comment by CaldwellYSR** [comment]
Oh I misunderstood the problem. I'll get that fixed tomorrow.

**Pull request merged**